components/collapsible-panel/collapsible-panel.js

```diff
@@ -0,0 +1,405 @@
+import '../button/button.js';
+import '../colors/colors.js';
+import '../icons/icon.js';
+import '../icons/icon-custom.js';
```

Comment:
I don't think you need `button.js` or `icon.js`.

Reply:
Yup, remnants of a past implementation